pkgs/development/tools/fusee-launcher/default.nix

```diff
@@ -0,0 +1,43 @@
+{ stdenv
+, lib
+, python3Packages
+, python3
+, fetchFromGitHub
+, pkgsCross
+, makeWrapper
+} :
+
+stdenv.mkDerivation rec {
+  name = "fusee-launcher-${version}";
+  version = "unstable-2018-11-14";
```

Comment:
> If a package is not a release but a commit from a repository, then the version part of the name must be the date of **that (fetched) commit**.

<cite>— [13.2. Package naming](https://nixos.org/nixpkgs/manual/#sec-package-naming)</cite>

Thus, the version should be *unstable-2018-07-14*.

Reply:
Whoops, you're right. Should be fixed now ;)